### ISSUE TITLE
Docs: Issues in Chapter 3 cont....

### ISSUE THREAD

**Issue by bsrikan** [marklogic/entity-services#280]
Issue1:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_92216
Actual:
The 'id' and 'name' properies are required.
Expected:
The 'id' and 'name' properties are required. 

Issue2:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_58667
Unbold 'x' in range-index for below text:
`include a rangeIndex JSON property or range-index XML element`

Issue3:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_47903
JSON descriptor example has "A" instead of "properties"

Issue4:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_21964
IRI prefix example cited doesnt match model descriptor as the document suggests.
`http://my/org/example-person/Person-1.0.0` 
There is no example-person or Person entity type in the example descriptor referenced just above this IRI in the docs. Maybe it should look like `http://my/org/Example-1.0.0` ie `baseUri/title-version` and `baseUri/title-version/EntityType` etc

Issue5:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_90749
Actual:
This is the $ref JSON property or ref XML element of the property definition.
Expected:
This is the $ref JSON property or es:ref XML element of the property definition.

Issue6:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_71069
Description for "Name" entity type should change to `The name of a person.` for XML descriptor.

Issue7:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_90749
You did say "might look like" in the docs so I guess its ok, otherwise Person entity instance actually looks like 
`{
"$type": "Person",
    "id": 1234,
    "name": {
      "$type": "Name",
      "first": "John",
      "middle": "NMI",
      "last": "Smith"
    }
} `

Issue8:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_28304
Looks like incomplete sentence.
"You can extend your model with information and relationships that cannot be expressed in and derived from the model descriptor by storing additional semantic triples related` to your in MarkLogic.`"

Issue9:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_31275
Actual:
However, both templates will use the same same row 
Expected:
However, both templates will use the same row 

Issue10:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_73175
Incomplete sentence
`Also, if you a range index or word lexicon`

Issue11:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_44223
TDE does not allow '-' and will throw an error when trying to insert a template via tde:insert(). It might be a nice idea to mention that for title description.

Issue12:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_19553
Contradictory. 1 vs 0 entity property. property_definition also says  "Each entity type must include at least one entity property definition.":
`An entity type definition must contain a properties XML element or JSON property that defines at least one entity property`
vs below in properties description
`Zero or more entity property definitions.`

Issue13:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_19553
Actual:
Each child JSON property or XML element name is the name of an property of the entity type.
Expected:
Each child JSON property or XML element name is the name of a property of the entity type.

Issue14:
http://pubs.marklogic.com:8011/9.0/guide/entity-services/models#id_13962
Sentence needs correction and punctuation(,) maybe.
`Entity property definition are be used in the properties child of an entity_type_definition.`

**Comment by bsrikan** [comment]
Kim seems to have pushed fixes for the issues in this bug. Updates are in Bugtrack # 45257. Assigning to self with label test.

**Comment by bsrikan** [comment]
Issue#7 still seems an issue. There was a code change after the docs example would have been written. What I have pasted is the current output.

**Comment by kcoleman-marklogic** [comment]
Sorry, I think this is another place where I tricked you by fixing the problem a different way. :) I should have mentioned it. 

Rather than changing the example output, I changed the description of what that output is. It now says:

>For example, a Person entity instance generated by es:instance-json-from-document or es.instanceJsonFromDocument might look like the following:

Previously, it said `es:instance-from-document` (and `es.instanceFromDocument`). 

I find the intermediate, almost internal, canonical representation produced by instance-from-document to be less interesting than the XML or JSON one might hand to downstream consumers of the data, so I thought it better to correct the description than change the data.
